packages/gr-sheet/src/index.ts

```diff
@@ -26,6 +26,7 @@ import { teeAsync } from '../../common/src/util.js';
 export const ParameterType = {
   FILE: 'parameter file name',
   MEASURE: 'measure (w/ UoM)',
+  INTEGER: 'integer value',
```

Comment:
@ribose-jeffreylau take a look at this: https://github.com/paneron/extension-geodetic-registry/blob/1366ad19d6cacf4ae571cf01241a12bf7ad6ca4d/src/classes/transformation.tsx#L36

The actual value of the constant looks right, but could you perhaps rename the constant to match `INTEGER_VALUE` for consistency?

Reply:
It is unfortunate that we cannot import it properly, it may be related to the same packaging issue that causes the TSC errors you encountered earlier.

Reply:
I can do that.